vulnerabilities/helpers.py

```diff
@@ -164,3 +165,34 @@ def __lt__(self, other):
         )
 
     return affected_package_with_patched_package_objects
+
+
+def split_markdown_front_matter(lines: str) -> Tuple[str, str]:
+    r"""
+    Split text into markdown front matter and the markdown body
```

Comment:
```suggestion
    Return a tuple of (front matter, markdown body) strings split from ``text``.
    Each can be an empty string.
```
(I use text rather than lines as a variable name)

Reply:
fixed